### DISCUSSION TITLE
Item swap feature

### DISCUSSION THREAD

**Discussion by SaladDressings** [xivdev/Penumbra#202] [Q&A]
Hiya ^^

I know it has been asked before, but is it possible now to change the target item that gets replaced? I noticed the Item swap tab under advanced settings but couldn't figure it out. Could someone explain to me what it does and how it works?

Thank you!

**Comment by Microondas4269** [comment]
this is actually a really good function, i struggle too much trying to match my mods to my glamours and then exporting as a modpack via textools and then loading into penumbra, takes tooo much time

**Reply by SaladDressings** [reply]
What I do rn I import the mod into textools and then use the Item converter tool to copy the modded items to one single set of useless gear. I make a mod from the transfer and put that into penumbra. Basically changes the target item. Then I just equip that useless gearset and play dress up (enabling/disabling all the mods).

**Reply by Microondas4269** [reply]
yea, that's literally what i do, it would be awesome if penumbra did the same thing we have to do in textools beforehand

**Comment by eclipsaa** [comment]
Im here for this exact subject.   I would rather NOT use texttools to double up mods.

**Comment by tancred423** [comment]
Item Swap does that job. Quick step by step instruction:

1. Select your mod in Penumbra and open "**Advanced Editing**".
2. Select "**Item Swap**".

> If you want to swap a mod within its slot, e.g. a hat to a different hat:
3. Select the tab with your desired slot, e.g. "**Hat**".
4. On "**Take this [...]**", select the item the mod currently replaces.
5. On "**and put it on this one**", select your target item.

> If you want to swap a mod between slots, e.g. a hat to an earring:
3. Select the "**Between Slots**" tab.
4. On "**Take these**", select the slot and item the mod currently replaces.
5. On "**and put them on this**", select your target slot and item.

> 
6. Click "**Create New Mod**".
7. Back in the normal Penumbra window, activate your mod with "**(Swapped)**" in its name and disable the original one.
8. Maybe "**Redraw**" yourself at the bottom of the window and then equip the correct items :D

**Reply by Panthanaea** [reply]
Is there a way to move say a top to a ring?

**Reply by zebrapurple** [reply]
I know this post is two and a half years old, but I'm struggling to figure out how to dye the item I want to show. I've got the items themselves swapped, but when I put the target (item I want to see) item's material file into materials and pick a dye, nothing happens when I press Apply preview dye. Dyeable is checked, and these are both base-game items, not mods. 

I should note I'm trying to make an outfit edit for Urianger, just by putting some different items on him, then I want to dye them black. Maybe the problem is that it's not the player character?

**Reply by Cyan101** [reply]
This just seems to move the old item over on some mods.

**Comment by Ottermandias** [comment]
This thread is more than a year old, but thank you for the detailed explanation. Also, why the fuck did I get a mail about this message just now when it is 5 days old.
